### ISSUE TITLE
Tool Names in Github Copilot Chat of VSCode Not Read Out Completely By Screen-Readers: A11y_VSCode_ScreenReader.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#273031]
### GitHub Tags:
#A11yTCS; #A11yMAS;#Visual Studio Code Client; #BM_VisualStudioCodeClient_Win32_JULY2024; #DesktopApp; #Win32;#Benchmark;#A11ySev2;#WCAG1.3.1;#Info and Relationship;#A11yeDAD;

Environment and OS details:
Application: Java Upgrade for VS Code
Version: 1.105.0-insider (user setup)
OS build :26100.4770

### Repro Steps:

1. Open GitHub copilot chat.
2. Github copilot screen will open.
3. Tab till GitHub copilot chatbot and ask something with #
4. Verify that Tool Names in Github Copilot Chat of VSCode Not Read Out Completely By Screen-Readers

### Actual Result:
Screen reader does not announce the context information along with the name of the list item. Currently, JAWS announces "number changes, tool".

### Expected Result:
screen reader should announce the complete context information along with the name of the list item. JAWS should announce "number changes, tool, Built-In: Git Changes"

### User Impact:
Screen reader users will not get the information of the options functionality if more info is not announced

### Attachment:

https://github.com/user-attachments/assets/6343c5ce-735e-48c5-8ec5-b8813a4f2b08

**Comment by roblourens** [comment]
I think this would be a general suggest widget issue, and I think there is an issue for that? @meganrogge
